components/x-audio/bower.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "x-audio",
+  "main": "dist/Audio.es5.js",
```

Comment:
This could be the ES6 bundle as we always transform the contents of Bower components (Origami components are ES6)